benchmarks/distributed/rpc/rl/agent.py

```diff
@@ -73,7 +77,7 @@ def select_action_batch(agent_rref, observer_id, state):
         observer_id -= 2
 
         if self.pending_states == self.world_size - 2:
-            agent_latency_start = time.time()
+            self.agent_latency_start = time.time()
```

Comment:
@mrshenli can you please confirm agent latency is being calculated correctly

Reply:
Do you need a lock to prevent concurrent access? What if the boundary of scheduling unit is between line 79 and 80, there can be a race here, right?